rpm/phonesim.spec

```diff
@@ -48,8 +48,8 @@ mkdir -p %{buildroot}%{_datadir}/applications/
 cp -a %{SOURCE2} %{buildroot}%{_datadir}/applications/
 mkdir -p %{buildroot}%{_datadir}/phonesim/
 cp -a %{SOURCE3} %{buildroot}%{_datadir}/phonesim/
-mkdir -p %{buildroot}%{_libdir}/systemd/user/
-cp -a %{SOURCE4} %{buildroot}%{_libdir}/systemd/user/
+mkdir -p %{buildroot}/usr/lib/systemd/user/
```

Comment:
Please use %_userunitdir instead.

Reply:
Changed.

Reply:
Commit message doesn't reflect the change exactly anymore.

Reply:
Hmm. I think it does.

> Use correct path for systemd unit. They are located in /usr/lib/systemd
> regardless of device bitness. Use systemd macro for that.